lib/providers/circuit-breaker/dynamo.ts

```diff
@@ -57,20 +58,36 @@ export class DynamoCircuitBreakerConfigurationProvider implements CircuitBreaker
         const enabledEndpoints = endpoints.filter((e) => {
           return !(fillerTimestamps.has(e.endpoint) && fillerTimestamps.get(e.endpoint)!.blockUntilTimestamp > now);
         });
-        const disabledEndpoints = endpoints.filter((e) => {
-          return fillerTimestamps.has(e.endpoint) && fillerTimestamps.get(e.endpoint)!.blockUntilTimestamp > now;
-        });
+        const disabledEndpoints = endpoints
+          .filter((e) => {
+            return fillerTimestamps.has(e.endpoint) && fillerTimestamps.get(e.endpoint)!.blockUntilTimestamp > now;
+          })
+          .map((e) => {
+            return {
+              webhook: e,
+              blockUntil: fillerTimestamps.get(e.endpoint)!.blockUntilTimestamp,
+            };
+          });
 
         this.log.info({ num: enabledEndpoints.length, endpoints: enabledEndpoints }, `Endpoints enabled`);
         this.log.info({ num: disabledEndpoints.length, endpoints: disabledEndpoints }, `Endpoints disabled`);
 
-        return enabledEndpoints;
+        return {
+          enabled: enabledEndpoints,
+          disabled: disabledEndpoints,
+        };
       }
 
-      return endpoints;
+      return {
+        enabled: endpoints,
+        disabled: [],
+      };
     } catch (e) {
       this.log.error({ error: e }, `Error getting eligible endpoints, default to returning all`);
-      return endpoints;
+      return {
```

Comment:
nit: this could be put in a finally and remove the duplication. We can have the `disabledEndpoints` initialized to `[]` outside the try{